skimage/filters/thresholding.py

```diff
@@ -1,10 +1,15 @@
+import math
```

Comment:
OK, doesn't make a huge difference, but say you have a function that's going to be called zillions of times, then you should really use ``math`` instead of ``numpy``.